### PR TITLE
Jose

### LINKED ISSUE #52: DFD 3 + Report

### LINKED ISSUE #57: DFD 3 Summary Of Observations

### PR DESCRIPTION
**Pull request by Jbibianochavez**

### CODE REVIEW

File: Threat Modeling.md [hunk 1]
```diff
@@ -46,6 +48,7 @@ TODO: Insert General Overall Summary Here
...
 
 ### Spoofing
+Home Assistant is relatively secure because users usually access the server on a local network. As a result, an attacker would need access to the local network before attempting to circumvent security. However, Home Assistant has experienced spoofing vulnerabilities in the past. It has been possible to use the ["X-Forwarded-For” option to spoof the IP address](https://github.com/home-assistant/core/issues/14345#issuecomment-401324487) to one that is on the trusted list. This vulnerability allowed attackers to bypass authentication once on the network and change the IP address once one is banned in SSH brute force attempts.  The vulnerability was mainly an issue for users using a reverse proxy and trusted network simultaneously in their set up. Home Assistant specified the dangers of using these settings together, but users did not read the warning in some cases. [Changes have been made](https://github.com/home-assistant/core/pull/15204) to the code to address these issues. When running home Assistant, it is crucial to ensure the local network is secured and only trusted users have access. Securing the local network will help prevent spoofing.
```
Comment (Chrs987) [inline Threat Modeling.md]: The sentence towards the end When running **home** Assistant just capitalize the H in Home Assistant

**Review by Chrs987 — DISMISSED** [review]
Excellent write up, once you correct the capitalization of the "H" in Home Assistant in the comment it is good on my end.

File: Threat Modeling.md (hunk 1 above)
Comment (Jbibianochavez) [reply Threat Modeling.md]: The correction has been made, thank you!

**Comment by Chrs987** [comment]
Excellent, thank you! Approved

**Review by RocketSockz — APPROVED** [review]

**Review by Chrs987 — APPROVED** [review]
Somehow I closed out the PR but I re-opened it and approved

**Pull request merged**